pkgs/development/libraries/libime/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, boost
+, python3
+, fcitx5
+}:
```

Comment:
as in Fedora, it maybe need eigen3, zlib, liblzma, bzip2 or some of them (package in Fedora)

Reply:
Maybe related to https://github.com/NixOS/nixpkgs/pull/104658#issuecomment-732069938

Reply:
pretty sure missing dependencies will cause the build to fail in the configure phase.